### ISSUE TITLE
pywattbox update breaks hass-wattbox

### ISSUE THREAD

**Issue by kbarnes3** [eseglem/hass-wattbox#18]
## Version of the custom_component
v0.8.1

## Configuration
Any configuration

## Describe the bug
The latest pywattbox releases on pypi make a breaking change that causes this integration to fail to init. Compare pywattbox's `__init__.py` between [v0.4.0](https://github.com/eseglem/pywattbox/blob/v0.4.0/pywattbox/__init__.py) and [v0.7.2](https://github.com/eseglem/pywattbox/blob/v0.7.2/pywattbox/__init__.py) to see the break.

I worked around this locally by pinning the pywattbox dependency to 0.4.0 exactly.

## Debug log
```text

Logger: homeassistant.setup
Source: setup.py:288
First occurred: 5:42:51 PM (1 occurrences)
Last logged: 5:42:51 PM

Error during setup of component wattbox
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 288, in _async_setup_component
    result = await task
             ^^^^^^^^^^
  File "/config/custom_components/wattbox/__init__.py", line 76, in async_setup
    from pywattbox import WattBox  # pylint: disable=import-outside-toplevel
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ImportError: cannot import name 'WattBox' from 'pywattbox' (/usr/local/lib/python3.11/site-packages/pywattbox/__init__.py)

```

**Comment by eseglem** [comment]
Sorry about that. I'll try to push out a new release pinning the version lower. 

Alternatively, you can also install the beta release, which works with it.

**Comment by eseglem** [comment]
This should be fixed by v0.8.2. If not, feel free to re-open the issue.
